src/AliasStorage.php

```diff
@@ -46,8 +46,12 @@ public function setDomainId($domain_id) {
   public function getDomainId() {
     // If no domain id has been set, use the currently active one.
     if (is_null($this->domain_id)) {
-      $this->domain_id = (int) $this->domain_negotiator->getActiveDomain()->getDomainId();
+      if (!is_null($this->domain_negotiator->getActiveDomain())) {
+        $this->domain_id = (int) $this->domain_negotiator->getActiveDomain()
```

Comment:
Why are we setting the value in the getter? This is causing all kinds of issues.